core/trino-main/src/main/java/io/trino/operator/aggregation/AggregationFromAnnotationsParser.java

```diff
@@ -127,20 +130,25 @@ private static AggregationHeader parseHeader(AnnotatedElement aggregationDefinit
     {
         AggregationFunction aggregationAnnotation = aggregationDefinition.getAnnotation(AggregationFunction.class);
 
-        return getNames(toParse, aggregationAnnotation).stream()
-                .map(name ->
+        return getNames(toParse, aggregationAnnotation).entrySet().stream()
+                .map(nameEntry ->
                         new AggregationHeader(
-                                name,
+                                nameEntry.getKey(),
+                                nameEntry.getValue(),
                                 parseDescription(aggregationDefinition, toParse),
                                 aggregationAnnotation.decomposable(),
                                 aggregationAnnotation.isOrderSensitive(),
                                 aggregationAnnotation.hidden()))
                 .collect(toImmutableList());
     }
 
-    private static List<String> getNames(@Nullable AnnotatedElement outputFunction, AggregationFunction aggregationAnnotation)
+    private static Map<String, String> getNames(@Nullable AnnotatedElement outputFunction, AggregationFunction aggregationAnnotation)
     {
-        List<String> defaultNames = ImmutableList.<String>builder().add(aggregationAnnotation.value()).addAll(Arrays.asList(aggregationAnnotation.alias())).build();
+        Map<String, String> defaultNames = ImmutableMap.<String, String>builder()
+                .put(aggregationAnnotation.value(), aggregationAnnotation.value())
+                .putAll(Arrays.asList(aggregationAnnotation.alias()).stream().collect(
+                        Collectors.toMap(k -> k, v -> aggregationAnnotation.value())))
```

Comment:
```suggestion
                .putAll(Stream.of(aggregationAnnotation.alias())
                        .collect(toImmutableMap(identity(), value -> aggregationAnnotation.value())))
```

Similarly in L162

Reply:
We still need to keep this line.
``` java
.put(aggregationAnnotation.value(), aggregationAnnotation.value())
```
Otherwise we will miss parsing function with the resolved name.

Reply:
Yeah !! I accidentally copied that line